testsuites/syncgateway/functional/tests/test_sgw_config.py

```diff
@@ -268,11 +269,14 @@ def test_invalid_external_jspath(params_from_base_test_setup, setup_jsserver):
 
     try:
         requests.get(sg_url, timeout=30)
-        assert False, "Sync gateway started successfully with invalid external jsfile "
```

Comment:
What is advantage of making this change.

Reply:
if the assert Fails, it is caught in exception rathar than showing that assertion is failed. With this change, this will catch only the exception with request.get, but not for assert False